docs/specification.md

```diff
@@ -579,7 +578,7 @@ Parameter (in Michelson):
 ```
 
 - The proposal is saved under `BLAKE2b` hash of proposal value and sender.
-- The `Natural` value: `proposalTokenAmount` determines how many sender's frozen tokens will be staked in addition to the [fee](#set_fixed_fee_in_token).
+- The `Natural` value: `proposalTokenAmount` determines how many sender's frozen tokens will be staked in addition to the fee.
```

Comment:
With the removal of this entrypoint and the `proposer_fixed_fee_in_token` (as suggested in a previous comment) there is no longer any documentation about the fixed fee.

Note that the `config` type in the `Configuration` section needs to be updated as it now contains a field for the fee, I suggest moving the comment that is now under `proposer_fixed_fee_in_token` there and to change the link here to point at that section.

Reply:
Have updated.